bootstrap/frontend/src/main/scala/io/udash/bootstrap/datepicker/UdashDatePicker.scala

```diff
@@ -0,0 +1,373 @@
+    * @param defaultDate Sets the picker default date/time. Overrides `useCurrent`.
+    * @param disabledDates Disables selection of dates in the array, e.g. holidays.
+    * @param enabledDates Disables selection of dates NOT in the array, e.g. holidays.
+    * @param icons Change the default icons for the pickers functions.
+    * @param useStrict Defines if moment should use strict date parsing when considering a date to be valid.
+    * @param sideBySide Shows the picker side by side when using the time and date together.
+    * @param daysOfWeekDisabled Disables the section of days of the week, e.g. weekends.
+    * @param calendarWeeks Shows the week of the year to the left of first day of the week.
+    * @param viewMode The default view to display when the picker is shown.
+    *                 Note: To limit the picker to selecting, for instance the year and month, use format: `MM/YYYY`
+    * @param toolbarPlacement Changes the placement of the icon toolbar.
+    * @param showTodayButton Show the "Today" button in the icon toolbar.
+    * @param showClear Show the "Clear" button in the icon toolbar.
+    * @param showClose Show the "Close" button in the icon toolbar.
+    * @param widgetPositioning Position of datepicker widget.
+    * @param widgetParent On picker show, places the widget at the identifier object if the element has css position: 'relative'.
+    * @param keepOpen Will cause the date picker to stay open after selecting a date if no time components are being used.
+    * @param inline Will display the picker inline without the need of a input field. This will also hide borders and shadows.
+    * @param keepInvalid Will cause the date picker to not revert or overwrite invalid dates.
+    * @param ignoreReadonly Allow date picker show event to fire even when the associated input element has the `readonly="readonly"` property.
+    * @param allowInputToggle If `true`, the picker will show on textbox focus and icon click when used in a button group.
+    * @param focusOnShow If `false`, the textbox will not be given focus when the picker is shown
+    * @param enabledHours Will allow or disallow hour selections.
+    * @param disabledHours Will allow or disallow hour selections.
+    * @param viewDate This will change the viewDate without changing or setting the selected date.
+    * @param tooltips This will change the tooltips over each icon to a custom string.
+    */
+  case class DatePickerOptions(format: String,
+                               dayViewHeaderFormat: String = "MMMM YYYY",
+                               extraFormats: Seq[String] = Seq.empty,
+                               stepping: Int = 1,
+                               minDate: Option[ju.Date] = None,
+                               maxDate: Option[ju.Date] = None,
+                               useCurrent: Boolean = true,
+                               collapse: Boolean = true,
+                               locale: Option[String] = None,
```

Comment:
We could check for udash.i18n usage&locale by default to derive the right locale and improve udash modules interop.